.env

```diff
@@ -0,0 +1,3 @@
+VITE_PORT = "8080"
+
+VITE_WEBSOCKET_URL = "ws://0.0.0.0:8179/ws/conference/"
```

Comment:
It's not just the websocket url that needs to be specified. Web app also needs to know the URL of the API endpoint

Reply:
Will be fixed